hrcareer/js/src/hrcareer/app.js

```diff
@@ -1,18 +1,28 @@
 /* eslint-env amd */
 
-define(function () {
+define(function (CRM) {
   'use strict';
 
-  CRM.$(function ($) {
-    $('body').on('crmFormLoad', function (event) {
-      if (event.profileName === 'hrcareer_tab') {
-        var accessName = $('[data-crm-custom="Career:End_Date"]').attr('name');
+  CRM.$('body').on('crmFormLoad', function () {
```

Comment:
Please create functions to make the code more readable.
Something like
```javascript
CRM.$('body').on('crmFormLoad', function () {
  if(isCareerTab()) {
    addHelpIcon();
  }
})
```

Similarly do this for the other tab also.